core/db/migrate/20150609093816_increase_scale_on_pre_tax_amounts.rb

```diff
@@ -0,0 +1,16 @@
+class IncreaseScaleOnPreTaxAmounts < ActiveRecord::Migration
+  def change
+    # set pre_tax_amount on shipments to discounted_amount - included_tax_total
+    # so that the null: false option on the shipment pre_tax_amount doesn't generate
+    # errors.
+    #
```

Comment:
I prefer what is there. This is a migration where the table names are already hardcoded.